experiments/data/merge_epmc_to_jsonl.py

```diff
@@ -0,0 +1,50 @@
+"""
+This file is specifically for taking a nested folder of jsonlines files 
+and merging them into one complete jsonlines file.
+
+e.g. 
+<dir>/2022_05_27/file1.jsonl
+<dir>/2022_05_25/file2.jsonl
+...
+"""
+from typing import List
+import os
+import jsonlines
+import multiprocessing
+from tqdm import tqdm
+
+# NOTE hardcoded paths
+ROOT = "/fsx/proj-chemnlp/jeb/europmc_deduped"
+OUT_DIR = "/fsx/proj-chemnlp/jeb/europmc_deduped/merged_paper_and_abstracts"
```

Comment:
will we move it to s3 after we have written it here?

Reply:
yes, I'm just keeping it there until we've tokenised it 👍